### DISCUSSION TITLE
Mapper Naming 관련

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#99] [Convention]
제인이 작업한 것 중에
`SocialAuthenticationResponse gitHubUserResponseToSocialAuthenticationResponse(GitHubUserResponse gitHubUserResponse, OAuthType type);`

이런게 있었어요
혹시 너무 길어보이면 이렇게 하는건 어떨까요?

1. Mapper는 리턴타입으로 이름을 짓는다 e.g. `SocialAuthenticationResponseMapper`
2. 메소드 네이밍은 매개변수를 이용하여 e.g. `fromXXX`와 같이 짓는다.

**Comment by janeljs** [comment]
저는 좋아요! 그런데 원래 이렇게 하려다가 프레디가 어떤 이유로 안된다고 하시지 않으셨나요?

**Reply by Dae-Hwa** [reply]
맞아요 메소드가 겹칠 수 있는데,
대신 리턴타입은 반드시 클래스명에 명시된 타입으로만 해주면 문제 없을 것 같아요

**Comment by janeljs** [comment]
메서드 이름은 map으로 결정
맵퍼 리턴타입은 맵퍼 인터페이스 타입과 동일해야 한다.
